src/components/Capture/index.js

```diff
@@ -27,6 +30,30 @@ export default class Capture extends Component {
 
   componentDidMount () {
     events.on('onMessage', (message) => this.handleMessages(message))
+    this.checkWebcamSupport();
+  }
+
+  checkWebcamSupport () {
+    DetectRTC.load( _ => {
+      this.setState({
+        DetectRTCLoading: false,
+        hasWebcam: DetectRTC.hasWebcam
+      })
+    });
+  }
+
+  supportsWebcam (){
+    if (DetectRTC.isGetUserMediaSupported && this.state.DetectRTCLoading) return true;
+    return this.state.hasWebcam;
+  }
+
+  //Fired when there is an active webcam feed
+  onUserMedia() {
+    this.setState({
+      hasWebcam: true,
+      hasWebcamPermission: true,
+      DetectRTCLoading: false
+    })
```

Comment:
Minor comment, but the semicolon usage is pretty inconsistent in these repos atm.

Do you have a backlog task to add linting into the build? The longer we wait, the more painful it will be